pulser-core/pulser/backend/qpu.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Defines the backend class for QPU execution."""
+from __future__ import annotations
+
+from typing import cast
+
+from pulser import Sequence
+from pulser.backend.remote import JobParams, RemoteBackend, RemoteResults
+from pulser.devices import Device
+
+
+class QPUBackend(RemoteBackend):
+    """Backend for sequence execution on a QPU."""
+
+    def run(self, job_params: list[JobParams] = []) -> RemoteResults:
+        """Runs the sequence on the remote QPU and returns the result.
+
+        Args:
+            job_params: A list of parameters for each job to execute. Each
+                mapping must contain a defined 'runs' field specifying
+                the number of times to run the same sequence. If the sequence
+                is parametrized, the values for all the variables necessary
+                to build the sequence must be given in it's own mapping, for
+                each job, under the 'variables' field.
```

Comment:
Yes, I chose to do it in `PasqalCloud.submit()` because I figured it could be specific to the cloud provider, but I can move the check on 'runs' here.